docs/30.egopulse/config.md

```
@@ -186,20 +189,39 @@ config.default_provider（グローバルプロバイダー）
-|---|---|---|
-| `EGOPULSE_LOG_LEVEL` | `log_level` | `string` |
-| `EGOPULSE_WEB_HOST` | `channels.web.host` | `string` |
-| `EGOPULSE_WEB_PORT` | `channels.web.port` | `u16` |
-| `EGOPULSE_WEB_ENABLED` | `channels.web.enabled` | `bool` |
-| `EGOPULSE_WEB_AUTH_TOKEN` | `channels.web.auth_token` | `string` |
-| `EGOPULSE_WEB_ALLOWED_ORIGINS` | `channels.web.allowed_origins` | `string`（カンマ区切り） |
-| `EGOPULSE_DISCORD_BOT_TOKEN` | `channels.discord.bot_token` | `string` |
-| `EGOPULSE_TELEGRAM_BOT_TOKEN` | `channels.telegram.bot_token` | `string` |
+### 4.1 読み込み優先順位
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)